### PR TITLE
Cleans up frontend

### PR DESCRIPTION
**Pull request by fzuljevic**

### CODE REVIEW

**Comment by dompagoj** [comment]
Komentar

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,26 +1,3 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export const App =() =>{
```
Comment (dompagoj) [inline frontend/src/App.tsx:1]: Very nice

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { App } from './App';
```
Comment (dompagoj) [inline frontend/src/index.tsx:3]: Very najs again

File: frontend/src/App.tsx (hunk 1 above)
Comment (fzuljevic) [reply frontend/src/App.tsx:1]: thx daddi

File: frontend/src/index.tsx (hunk 2 above)
Comment (dompagoj) [reply frontend/src/index.tsx:3]: what

**Review by dompagoj — CHANGES_REQUESTED** [review]

**Review by dompagoj — APPROVED** [review]

**Pull request merged**